src/components/Cart.jsx

```diff
@@ -0,0 +1,101 @@
+import PropTypes from 'prop-types';
+
+import CartItem from './CartItem';
+import { Button } from './UI';
+
+const Cart = ({ cart, tickets, items, teamMembers, onItemRemoved, onTicketRemoved, onCartReset }) => {
+  let attendantTicket = items.find((ticket) => ticket.id === 'ticket-attendant');
+  return (
+    <div className="cart">
+      <div className="cart-header">
+        <h2>Panier</h2>
+        <Button onClick={onCartReset} rightIcon="fas fa-trash-alt red-icon" noStyle />
+      </div>
+      {Object.entries(tickets).map(([userId, ticket]) => {
+        let i = cart.tickets.userIds.findIndex((id) => id === userId);
+        let user = teamMembers.find((user) => user.id === userId);
+        return (
+          <CartItem
+            key={`${ticket.id}|${userId}`}
+            itemName={`${ticket.name} (${user.username})`}
+            quantity={1}
+            unitPrice={ticket.price}
+            reducedUnitPrice={ticket.reducedPrice}
+            onRemove={() => onTicketRemoved(user, i)}
+          />
+        );
+      })}
+      {cart.tickets.attendant && (
+        <CartItem
+          key={attendantTicket.id}
+          itemName={`${attendantTicket.name} (${cart.tickets.attendant.firstname} ${cart.tickets.attendant.lastname})`}
+          quantity={1}
+          unitPrice={attendantTicket.price}
+          onRemove={() => onTicketRemoved(undefined, undefined)}
+        />
+      )}
+      {cart.supplements.map((supplement) => {
+        let item = items.find((item) => item.id == supplement.itemId);
+        return (
+          <CartItem
+            key={item.id}
+            // NOTE : For the moment, only t-shirts have attributes, but that may change in the future
+            itemName={item.name + (item.attribute ? `- Taille ${item.attribute.toUpperCase()}` : '')}
```

Comment:
Tu peux ajouter un petit espace avant ^^
```suggestion
            itemName={item.name + (item.attribute ? ` - Taille ${item.attribute.toUpperCase()}` : '')}
```

Reply:
aoé